code-samples/keyv/index.js

```diff
@@ -32,13 +32,13 @@ client.on('messageCreate', async message => {
 
 	const command = args.shift().toLowerCase();
 
-	if (command === 'prefix') {
+	if (command === 'prefijo') {
```

Comment:
```suggestions
if (command === 'prefix') {
```
En el codigo la variable se llama asi